api/CacheAccess.php

```diff
@@ -1,21 +1,49 @@
 <?php
 
-class CacheAccess
-{
+class CacheAccess {
 
     /**
      * Loads html from url or cache, if present.
      * @param $url
      * @param int $ttl
      * @return false|file_get_html|mixed
      */
-    static function getHtml($url, $ttl = 60){
-        if(apcu_exists($url)){
+    static function getHtml($url, $ttl = 60) {
+        if (apcu_exists($url)) {
             $html = apcu_fetch($url);
```

Comment:
Can you convert the URLs here please and store the right HTML in the cache with the short links? Then recomputing will take less of a hit on refreshes.

Reply:
I hope my changes are about what you thought of :)